.github/workflows/publish.yml

```
@@ -0,0 +1,45 @@
+name: Publish
+
+# pypi trusted publishing via OIDC
+permissions:
+  id-token: write
+
+on:
+  pull_request:
+    branches: [ main, dev ]
```

```suggestion
    branches: [ master ]
```

Right? No main or dev branch exist

---

habit. good catch!